From c36b42456de58d8ebc852fb13261d8510a5cbc4b Mon Sep 17 00:00:00 2001
From: Dimitar Dimitrov <dimitar@dinux.eu>
Date: Mon, 2 Jan 2017 10:20:27 +0200
Subject: [PATCH 3/9] Disable sleb128-4 for PRU

TODO - check if this is a core issue, or a PRU port issue.

Signed-off-by: Dimitar Dimitrov <dimitar@dinux.eu>
---
 gas/testsuite/gas/all/sleb128-4.d | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gas/testsuite/gas/all/sleb128-4.d b/gas/testsuite/gas/all/sleb128-4.d
index 89b9565301..283d946432 100644
--- a/gas/testsuite/gas/all/sleb128-4.d
+++ b/gas/testsuite/gas/all/sleb128-4.d
@@ -1,6 +1,6 @@
 #objdump : -s -j .data -j "\$DATA\$"
 #name : .sleb128 tests (4)
-#skip: msp430*-*-*
+#skip: msp430*-*-* pru*-*-*
 
 .*: .*
 
-- 
2.11.0

